hugo/layouts/index.html

```diff
@@ -52,168 +57,58 @@ <h2>Explore species</h2>
                     </a>
-
-                    {{ if .Params.img_attrib_link }}
-                    <a href="{{ .Params.img_attrib_link }}" title="Go to the image source" target="_blank">
-                        <figcaption class="scilife-card-image-attrib"> {{ .Params.img_attrib_text }} </figcaption>
+                    <!-- Include format for with and without link, JS decides which to display.-->
+                    <a class="caption-with-link" href="#" title="Go to the image source" target="_blank" style="display: none;">
+                        <figcaption class="scilife-card-image-attrib"></figcaption>
                     </a>
-                    {{ else }}
-                    <p class="scilife-card-image-attrib">
-                        {{ .Params.img_attrib_text }}
-                    </p>
-                    {{ end }}
-
+                    <p class="caption-no-link scilife-card-image-attrib" style="display: none;"></p>
                 </figure>
             </div>
 
             <div class="card-body">
-                <h2 class="card-title" id="science-name">{{ .Title | markdownify }}</h2>
-                <h3 class="card-title" id="common-name">{{ .Params.subtitle | markdownify }}</h3>
+                <h2 class="card-title science-name"></h2>
+                <h3 class="card-title common-name"></h3>
             </div>
 
             <div class="d-flex justify-content-between align-items-center">
-                <p class="card-text mb-0 mt-auto">Last updated: {{ .Params.last_updated }}</p>
-                <!-- The date below is hidden because used only in the sorting algo for easier date parsing -->
-                <div class="hidden-date" style="display: none;">{{ .Params.last_updated }}</div>
-
-                <a href="{{ .RelPermalink }}" class="btn btn-primary scilife-species-button"
-                    title="Go to the {{ .Title | markdownify }} page">
+                <p class="card-text mb-0 mt-auto"></p>
+                <a href="#" class="btn btn-primary scilife-species-button"title="">
                     <i class="bi bi-arrow-right" style="font-size: 1.25em;"></i>
                 </a>
             </div>
 
         </div>
     </div>
     {{ end }}
-
 </div>
 
-<!-- shows if no search results -->
+
+<!-- shows only if no search results -->
 <div id="no-filtered-card" class="alert alert-primary mt-3" style="display: none;" role="alert">
     No results found with your search term
 </div>
 
 
+<!-- Add pagination if numb of species cards is > than number of species per page
+     numb species per page set in main config.yaml file -->
+{{if gt $numbSpecies $numbSpeciesCards }}
+    {{ partial "pagination.html" (dict "numbSpecies" $numbSpecies "numbSpeciesCards" $numbSpeciesCards) }}
+{{end}}
+
+
+{{ $speciesCardsJS := resources.Get "js/species_cards.js" | fingerprint | minify }}
+<script type="module" src="{{ $speciesCardsJS.RelPermalink }}"></script>
```

Comment:
```suggestion
<script src="{{ $speciesCardsJS.RelPermalink }}"></script>
```
This is a standalone script so I shouldn't have this here